website/src/components/CustomerLogos.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FunctionComponent } from 'react'
+
+interface Logos {
+    name: string
+    src: string
+}
+
+const logos: Logos[] = [
+    {
+        name: 'Amazon',
+        src: '/external-logos/amazon-logo.svg',
+    },
+    {
+        name: 'GE',
+        src: '/external-logos/general-electric-logo.svg',
+    },
+    {
+        name: 'Qualtrics',
+        src: '/external-logos/qualtrics-logo.svg',
+    },
+    {
+        name: 'Dropbox',
+        src: '/external-logos/dropbox-logo.svg',
+    },
+    {
+        name: 'Nutanix',
+        src: '/external-logos/nutanix-logo.svg',
+    },
+    {
+        name: 'Paypal',
+        src: '/external-logos/paypal-logo.svg',
+    },
+    {
+        name: 'Uber',
+        src: '/external-logos/uber-logo.svg',
+    },
+    {
+        name: 'Plaid',
+        src: '/external-logos/plaid-logo.svg',
+    },
+    {
+        name: 'Reddit',
+        src: '/external-logos/reddit-logo.svg',
+    },
+    {
+        name: 'Cloudflare',
+        src: '/external-logos/cloudflare-logo.svg',
+    },
+    {
+        name: 'Mercado Libre',
+        src: '/external-logos/mercado-libre-logo.svg',
```

Comment:
The SVGs for Reddit, Mercado Libre, and Cloudflare are not fully transparent (they still have areas of white within them). This is not noticeable on the home page, but slightly noticeable on pages like /use-cases, where we are using `alabaster` as the bg color. Fully transparent SVGs may be more flexible for future use. I've noted this below:

(on /use-cases)
<img width="600" alt="Screen Shot 2022-04-21 at 2 42 59 PM" src="https://user-images.githubusercontent.com/60713139/164796562-0b5a23cc-82fe-4457-9b7e-12f869f43e19.png">

(highlighted)
<img width="600" alt="Screen Shot 2022-04-21 at 12 34 46 PM" src="https://user-images.githubusercontent.com/60713139/164796397-c54b3926-c34b-4694-b239-90930437a6f4.png">

Reply:
I went ahead and fixed these in Figma as well as in the SVG's. The Reddit logo didn't have a proper clipping mask and the mercado and cloudflare logos had white bg fills on those paths. All removed now! Good eye on this! Now we can have backgrounds of all kinds to our hearts content. 🤓 

![Screen Shot 2022-04-22 at 5 49 35 PM](https://user-images.githubusercontent.com/1733936/164801281-cae6023a-031e-4666-b524-149ba1bce49a.png)

Reply:
Sweet!! This rocks, Brett!